Jenkinsfile

```diff
@@ -1,3 +1,15 @@
+def githubUsernameToSlackName(github_author) {
+  // Add team members as necessary
+  def mapping = [
```

Comment:
This is fine for now, but in the future, I think we could move this onto the cluster file-sytem